cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -63,8 +72,55 @@ def build_load_config(self) -> RarediseaseLoadConfig:
         self.include_pedigree_picture(load_config)
         self.load_custom_image_sample(load_config)
         load_config.human_genome_build = GenomeBuild.hg19
+        load_config.rank_score_threshold = RANK_MODEL_THRESHOLD
+        load_config.rank_model_version = self.get_rank_model_version(variant_type=Variants.SNV)
+        load_config.sv_rank_model_version = self.get_rank_model_version(variant_type=Variants.SV)
         return load_config
 
+    def get_rank_model_version(self, variant_type: Variants) -> str:
+        """
+        Returns the rank model version for a variant type from the manifest file.
+        Raises FileNotFoundError if no manifest file is found in housekeeper.
+        """
+        hk_manifest_file: File = self.get_file_from_hk({NFAnalysisTags.MANIFEST})
+        if not hk_manifest_file:
+            raise FileNotFoundError("No manifest file found in Housekeeper.")
+        return self.extract_rank_model_from_manifest(
+            hk_manifest_file=hk_manifest_file, variant_type=variant_type
+        )
+
+    def extract_rank_model_from_manifest(
+        self, hk_manifest_file: File, variant_type: Variants
+    ) -> str:
+        content: dict[str, dict[str, str]] = ReadFile.get_content_from_file(
+            file_format=FileFormat.JSON, file_path=hk_manifest_file
+        )
+        return self.get_rank_model_version_from_manifest_content(
+            content=content, variant_type=variant_type
+        )
+
+    def get_rank_model_version_from_manifest_content(
+        self, content: dict[str, dict[str, str]], variant_type: Variants
+    ) -> str:
+        """
+        Return the rank model version from the manifest file content.
+        """
+        pattern: str = variant_type.upper() + ":GENMOD_SCORE"
+        for key, value in content["tasks"].items():
+            process: str = value.get("process")
+            script: str = value.get("script")
+            if pattern in process and AnalysisTag.CLINICAL in script:
+                return self._get_version_from_manifest_script(script)
```

Comment:
add error raising as discussed

Reply:
Added and tested: correct behavior observed